homeassistant/components/russound_rio/media_player.py

```diff
@@ -215,3 +219,43 @@ async def async_mute_volume(self, mute: bool) -> None:
     async def async_media_seek(self, position: float) -> None:
         """Seek to a position in the current media."""
         await self._zone.set_seek_time(int(position))
+
+    @command
+    async def async_play_media(
+        self, media_type: MediaType | str, media_id: str, **kwargs: Any
+    ) -> None:
+        """Play media on the Russound zone."""
+
+        if media_type not in {RUSSOUND_MEDIA_TYPE_PRESET}:
+            raise HomeAssistantError(
+                translation_domain=DOMAIN,
+                translation_key="unsupported_media_type",
+                translation_placeholders={
+                    "media_type": media_type,
+                },
+            )
+        if media_type == RUSSOUND_MEDIA_TYPE_PRESET:
```

Comment:
Just do early return/exception here for now.

```
if media_type != ...
   raise ..

source_id = ...
```

Please don't "prepare" the code for additional media types. That will make review harder and add complexity until needed.